### PR TITLE
New Developer Blog Contribution Flow

### PR DESCRIPTION
**Pull request by jasnonaz**
## Description & motivation
We're publishing a new flow for contribution to the dbt Developer Blog. The goal here is to reduce the complexity and number of steps required for community contributors and to clarify the purpose of the Developer Blog for contributors. To that end we have a new issue template and contribution guide.

## To-do before merge
- [X] Create #community-writers channel on slack
- [x] Delete #i-want-to-write-about discussions section

### CODE REVIEW

File: contributing/developer-blog.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- ⛹🏼‍♂️ Developer blog content is written by and for *prac*ti*tioners -* end users of analytics tools (and sometimes people that work with practitioners). **
+- ⭐ Developer blog content is best when it is *the story which the author is uniquely positioned to tell.* Authors are encouraged to consider what insight they have that is specific to them and the work they have done.
+- 🏎️  Developer blog content is actionable - readers walk away with a clear sense of how they can use this information to be a more effective practitioner. Posts include code snippets, Loom walkthroughs and hands-on, practical information that can be integrated into daily workflows.
+- 🤏 Nothing is too small to share - what you think is simple has the potential to change someone's week.
+- 🔮 Developer blog content is present focused —posts tell a story of a thing that you've already done or are actively doing, not something that you may do in the future.
+- 🕸️ Developer blog posts build a knowledge graph. Knowledge of advanced concepts and technical skills can be assumed, but it is best to include many, densely linked references to other posts.
```
Comment (gwenwindflower) [inline contributing/developer-blog.md]: Some of this has shifted a bit as Guides has become clearer. We should perhaps rephrase to this to make it clear that DevHub as a whole is a knowledge graph and how blogs fit into that.

File: contributing/developer-blog.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### What if I need help / have questions:
+
+- Feel free to post any questions in #community-writers on the dbt Slack.
+
+## Core Principles
+
+- 🧑🏻‍🤝‍🧑🏾 The dbt Developer blog is written by humans  **- individual analytics professionals sharing their insight with the world. To the extent feasible, a community member posting on the developer blog is not staking an official organizational stance, but something that *they* have learned or believe based on their work. This is true for dbt Labs employees as well.
+- 💍 Developer blog content is information rich - these are posts that readers share, bookmark and come back to time and time again.
```
Comment (gwenwindflower) [inline contributing/developer-blog.md]:
```suggestion
- 💍 Developer blog content is knowledge rich - these are posts that readers share, bookmark and come back to time and time again.
```

File: contributing/developer-blog.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- ⛹🏼‍♂️ Developer blog content is written by and for *prac*ti*tioners -* end users of analytics tools (and sometimes people that work with practitioners). **
```
Comment (gwenwindflower) [inline contributing/developer-blog.md]:
```suggestion
- ⛹🏼‍♂️ Developer blog content is written by and for *practitioners* - end users of analytics tools (and sometimes people that work with practitioners).
```

File: contributing/developer-blog.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+We are very interested in stacks, new tools and integrations and will happily publish posts about this - with the caveat that the *focus* of the post should be solving real world problems. Hopefully if you are writing about these, this is something that you have used yourself in a hands on, production implementation.
+
+- **Posts Should Have Right Sized Scope**: We want to be able to cover a topic in-depth and dig into the nuances. Big topics like “How should you structure your data team” or “How to ensure data quality in your organization” will be tough to cover in the scope of a single post. If you have a big idea - try subdividing it! “How should you structure your data team” could become “How we successfully partnered with our RevOps team on improving lead tracking” and “How to ensure data quality in your organization” might be “How we cleaned up our utm tracking”.
```
Comment (gwenwindflower) [inline contributing/developer-blog.md]: I think of one of our (potentially undocumented) stylistic pushes is 'DRY Lists' -- 'Post should' twice indicates maybe that part could just be dropped?

File: contributing/developer-blog.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+The dbt Developer Blog is a place where analytics practitioners can go to share their knowledge with the community. Analytics Engineering is a discipline we’re all  building together. The developer blog exists to cultivate the collective knowledge that exists on how to build and scale effective data teams.
+
+### What makes a good developer blog post?
+
+- The short answer: Practical, hands on analytics engineering tutorials and stories
+    - [Slim CI/CD with Bitbucket](https://docs.getdbt.com/blog/slim-ci-cd-with-bitbucket-pipelines)
+    - [So You Want to Build a dbt Package](https://docs.getdbt.com/blog/so-you-want-to-build-a-package)
+    - [Founding an Analytics Engineering Team](https://docs.getdbt.com/blog/founding-an-analytics-engineering-team-smartsheet)
+- See the Developer Blog core principles
```
Comment (gwenwindflower) [inline contributing/developer-blog.md]: should this be a link?

File: contributing/developer-blog.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+### How do I submit a proposed post?
+
+To submit a proposed post, open a `Contribute to the dbt Developer Blog` issue on the [Developer Hub repo](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose). You will be asked for:
+
+- A short (one paragraph) summary of the post you’d like to publish
+- An outline of the post (example)
```
Comment (gwenwindflower) [inline contributing/developer-blog.md]: We should make this a link to an example before we publish or as part of this PR?

**Review by gwenwindflower — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/contribute-to-developer-blog.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: read-contribution
+    attributes:
+      label: I have read the dbt Developer Blog contribution guidelines.
+      description: You can the contribution guide [here](../../contributing/developer-blog.md)
+      options:
+        - label: I have read the dbt Developer Blog contribution guidelines.
```
Comment (KiraFuruichi) [inline .github/ISSUE_TEMPLATE/contribute-to-developer-blog.yml:26]:
```suggestion
        - label: I have read the dbt Developer Blog contribution guidelines.
        - label: I work for dbt Labs and am creating this issue for a marketing-approved piece.
```

add marketing option

File: .github/ISSUE_TEMPLATE/contribute-to-developer-blog.yml [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+      description: How can we get in touch with you?
+      placeholder: your preferred email and/or dbt Slack handle
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: read-contribution
+    attributes:
+      label: I have read the dbt Developer Blog contribution guidelines.
+      description: You can the contribution guide [here](../../contributing/developer-blog.md)
```
Comment (KiraFuruichi) [inline .github/ISSUE_TEMPLATE/contribute-to-developer-blog.yml]:
```suggestion
      description: You can find the contribution guide [here](../../contributing/developer-blog.md)
```
Missing the word 'find'

File: contributing/developer-blog.md (hunk 6 above)
Comment (jasnonaz) [reply contributing/developer-blog.md]: Ahh yes I was looking for a good example and couldn't find one. May drop the example for now and we can add in when we have a good example outline.

File: contributing/developer-blog.md (hunk 1 above)
Comment (jasnonaz) [reply contributing/developer-blog.md]: good call - dropped this for now and we can include in the section on guides when that is ready

File: contributing/developer-blog.md (hunk 4 above)
Comment (jasnonaz) [reply contributing/developer-blog.md]: ✔️

File: contributing/developer-blog.md (hunk 5 above)
Comment (jasnonaz) [reply contributing/developer-blog.md]: ✅

**Review by gwenwindflower — APPROVED** [review]
:shipit:

**Pull request merged**